lib/config-generator.js

```diff
@@ -77,7 +77,7 @@ class ConfigGenerator {
 
         config.resolve = {
             extensions: ['.js', '.jsx', '.vue', '.ts', '.tsx'],
-            alias: {}
+            alias: this.webpackConfig.aliases
```

Comment:
you should create a copy of it, so that modifying the generated webpack config object cannot modify the Encore config (which might be used again, even if it is probably only an edge case to reuse it without resetting).
this also applies to externals.

Reply:
Definitely, nice catch, I'll open a new PR for that.

I'm a bit worried that there is the same issue with other objects though... for instance when calling `Encore.addLoader()` and `Encore.addPlugin()`.

Reply:
Well, if we don't mutate the object ourselves, it might be less of an issue. but here, we are doing the mutation ourselves in some cases